### PR TITLE
cprnc: add new package

### PR DESCRIPTION
**Pull request by jedwards4b**
cprnc is a fortran based file comparison program for netcdf data files.

### CODE REVIEW

File: var/spack/repos/builtin/packages/cprnc/package.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+    depends_on("netcdf-fortran")
+    depends_on("cmake@3:", type="build")
+
+    def install(self, spec, prefix):
+        with working_dir("spack-build", create=True):
+            which("cmake")("..", *std_cmake_args)
+            make()
+            make("test")
+            make("install")
```
Comment (alalazo) [inline var/spack/repos/builtin/packages/cprnc/package.py]: Why is this necessary? Is it just to call `make("test")`?

**Review by alalazo — CHANGES_REQUESTED** [review]

File: var/spack/repos/builtin/packages/cprnc/package.py (hunk 1 above)
Comment (jedwards4b) [reply var/spack/repos/builtin/packages/cprnc/package.py]: No, cmake was not being invoked by default - I found this solution in an issue in the spack repo, I've lost track of the issue now but it was failing with version 0.21.0.dev0 of spack which is where I was testing should I remove it?   And if I remove it what is the proper way to add a make("test")?

Comment (alalazo) [reply var/spack/repos/builtin/packages/cprnc/package.py]: I think you can just delete this, and it should work automatically. By default, `make test`  is executed after the `build` phase if the `--test` flag is passed to `spack install`.

Comment (jedwards4b) [reply var/spack/repos/builtin/packages/cprnc/package.py]: Done

**Review by alalazo — APPROVED** [review]
